app/models/exercise/problem.rb

```diff
@@ -21,20 +21,15 @@ def reset!
     self.expectations = []
   end
 
+  alias_method :own_expectations, :expectations
+  alias_method :own_custom_expectations, :custom_expectations
```

Comment:
I was about to suggest the use of `revamp_accessor` here but it's better that we keep the original behavior in `own_custom_expectations`